### DISCUSSION TITLE
Reverse Magnetised Mode

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#18294] [Ideas] [answered]
Magnetise mode is good, adding the opposite of magnetise (making circles go away from your cursor) would be good too.

**Comment by peppy** [comment]
A "repel" checkbox or letting the magnetism go negative should work quite well.

**Comment by ggliv** [comment]
Simple enough to implement (at least at a rough level).

https://user-images.githubusercontent.com/47010459/170302911-90ad7d98-ec97-4a13-af97-9aea971460a3.mp4

I'm not going to make any new PRs until my current ones are taken care of, but I can confirm that this is fun and easy to add if anyone else wants to take a crack at it in the meantime.

**Reply by ghost** [reply]
Looks beautiful!

**Reply by peppy** [reply]
I don't think you should hold back from PRing separately as it's a pretty simple implementation, but that's up to you.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/18607
